versatile-core/src/main/java/io/github/nscuro/versatile/Constraint.java

```diff
@@ -95,6 +95,22 @@ boolean matches(final Version version) {
         };
     }
 
+  /**
+   * Inverts the comparator of the constraint e.g. < 1.3 becomes >= 1.3
```

Comment:
Please [format](https://reflectoring.io/howto-format-code-snippets-in-javadoc/) code containing `<` and `>` as they would otherwise cause JavaDoc generation to fail.

For example:

```
[...] {@code < 1.3} becomes {@code >= 1.3}
```